src/sys/unix.rs

```diff
@@ -539,7 +549,14 @@ impl RecvFlags {
     ///
     /// [`SEQPACKET`]: Type::SEQPACKET
     pub const fn is_end_of_record(self) -> bool {
-        self.0 & libc::MSG_EOR != 0
+        // TODO: Expose this constant in libc for the ESP-IDF/LwIP framework
+        #[cfg(target_os = "espidf")]
+        const MSG_EOR: libc::c_int = 8;
```

Comment:
I don't want to maintain these constants so please add them to libc instead.

Reply:
I think we might have alternatives here, let me do a little digging and get back to you with options.

Reply:
Oops yeah we can follow redox's example here and just conditionally remove the feature.  I'll work on that soon and update.  Thanks for the review!